packages/router-plugin/src/router.state.ts

```diff
@@ -118,8 +116,13 @@ export class RouterState {
     });
   }
 
-  private navigationEnd(): void {
-    this.routerStateSnapshot = this._serializer.serialize(this.lastResolvedStateSnapshot);
+  private guardsCheckEnd(routerState: RouterStateSnapshot): void {
+    if (this.routerStateSnapshot) {
+      // routerState = { ...this.routerStateSnapshot, ...routerState };
```

Comment:
Is it really worth leaving comments in the code?

Reply:
Ah, that was Artur's fix, but it turned out that it was not needed. I'll remove it.